lesson_19_2/test_app.py

```diff
@@ -0,0 +1,36 @@
+import os
+import requests
+import tempfile
+
+def test_upload_retrieve_image():
```

Comment:
It is not the first time I highlight that it is necessary to create test cases inside Test Class. That is best practice of Test Automation approach. Please fix that in your code.

Reply:
Added class